src/test/java/giis/eshopcontainers/e2e/functional/LoggedUserTest.java

```diff
@@ -0,0 +1,31 @@
+package giis.eshopcontainers.e2e.functional;
+
+import giis.eshopcontainers.e2e.functional.common.BaseLoggedTest;
+import giis.eshopcontainers.e2e.functional.common.exceptions.ElementNotFoundException;
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebElement;
+
+class LoggedUserTest extends BaseLoggedTest {
+    @Test
+    @DisplayName("BasicLoginTest")
+    void loginTest() throws InterruptedException, ElementNotFoundException {
+        log.info("Starting test");
```

Comment:
If you need log all start and end of tests, you can move logging to the base class (and include the test name)